core-api/README.md

```diff
@@ -0,0 +1,102 @@
+
+### ForwardingEventProcessor
+[`ForwardingEventProcessor`](https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/event/ForwardingEventProcessor.java)
+implements `EventProcessor` for backwards compatibility. Each event Processed is translated into a [`LogEvent`](https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/event/ForwardingEventProcessor.java)
+before getting synchronously sent along to the supplied `EventHandler`.
+
+## EventHandler
+[`EventHandler`](https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/event/EventHandler.java)
+interface is used for dispatching events to the Optimizely event end-point. Implementations of `dispatchEvent(LogEvent)` are expected
+to make an HTTP request of type `getRequestMethod()` to the `LogEvent#getEndpointUrl()` location. The corresponding request parameters and body
+are available via `LogEvent#getRequestParams()` and `LogEvent#getBody()` respectively.
+
+### NoopEventHandler
+[`NoopEventHandler`](https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/event/NoopEventHandler.java)
+implements `EventHandler` with no side-effects. Useful for testing or non-production environments.
+
+## NotificationCenter
+[`NotificationCenter`](https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/notification/NotificationCenter.java)
+is the centralized component for subscribing to notifications from the SDK. Subscribers must implement the `NotificationHandler<T>` interface
+and are registered via `NotificationCenter#addNotificationHandler`. Notifications are currently sent synchronously, so be sure that 
+implementations do not block unnecessarily.
+
+## ProjectConfig
+[`ProjectConfig`](https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/config/ProjectConfig.java)
+represents the current state of the Optimizely project as configured through [optimizely.com](https://www.optimizely.com/).
+The interface is currently unstable and used only internally. All public access to these implementations are subject to change
+with each subsequent version.
+
+### DatafileProjectConfig
+[`DatafileProjecyConfig](https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/config/DatafileProjectConfig.java)
+is an implementation of `ProjectConfig` backed by a file typically located within the Optimizely CDN.
+
+## ProjectConfigManager
+[`ProjectConfigManager`](https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/config/ProjectConfigManager.java)
+is a factory class that serves a `ProjectConfig`. Implementations of this class serve to provide a consistent representation
+of a `ProjectConfig` that can be passed between services.
```

Comment:
Suggest deleting "serve to" and just say "Implementations of this class provide..."